common/_webdriver_qa_api/mobile/utils.py

```diff
@@ -0,0 +1,113 @@
+import logging
```

Comment:
1) I thinks it's ok, because utils implements actions that related with mobile device but web_driver/mobile_driver modules have actions that related with WebDriver session/object  

2) I think that it almost the same usage becouse MobileDriver setting keep the same variables from configs and we can't use mobile part without this config variables (creation of mobile driver object based on it).

3) I think that config and WebDriver modules placed on same layer (top), is it not true?

Reply:
Discussed offline.
ToD: Use MobileDriver instance to get capabilities directly from the method